src/Groups/sub.jl

```diff
@@ -63,11 +67,11 @@ end
 """
     issubgroup(G::T, H::T) where T <: GAPGroup
 
-Return (`true`,`f`) if `H` is a subgroup of `G`, where `f` is the embedding homomorphism of `H` into `G`, otherwise return (`false`,`Nothing`).
+Return (`true`,`f`) if `H` is a subgroup of `G`, where `f` is the embedding homomorphism of `H` into `G`, otherwise return (`false`,`nothing`).
 """
 function issubgroup(G::T, H::T) where T <: GAPGroup
    if false in [h in G for h in gens(H)]
```

Comment:
I realize this code was not added in this PR, but let me point out that this is very inefficient, esp. if the number of generators is high: it has to create a vector of size equal to the number of generators, always looks at all generators, and has to search the whole generated vector; but in reality, a single bit as storage is sufficient, and we can abort immediately as soon as we find an element not in `G`.

Better alternative (and also more readable, IMHO)
```suggestion
   if !all(h -> h in G, gens(H))
```